install.sh

```
@@ -0,0 +1,1432 @@
+    local step_num="1"
+    [ "$DEPLOY_MODE" = "offline" ] && step_num="2"
+
+    echo -e "${BLUE}=========================================${NC}"
+    echo -e "${BLUE}🚀 Step ${step_num}: Kubespray 클러스터 배포${NC}"
+    echo -e "${BLUE}=========================================${NC}"
+    echo ""
+
+    # Harbor insecure registry extra-vars 구성
+    local harbor_extra_vars=""
+    if [ -n "$harbor_registry" ]; then
+        harbor_extra_vars="-e containerd_insecure_registries=['\"${harbor_registry}\"']"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)